package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "ember-lazy-video",
   "description": "Lazy loading videos for Ember apps",
-  "version": "0.2.5",
+  "version": "0.3.2",
```

Comment:
i think i completely forgot about this comment :P sorry i squashed commits and reverted the `version` to `0.2.5`